tests/e2e/etcd_config_test.go

```diff
@@ -302,6 +302,44 @@ func TestGrpcproxyAndCommonName(t *testing.T) {
 	}
 }
 
+func TestGrpcproxyAndListenCipherSuite(t *testing.T) {
+	skipInShortMode(t)
+
+	argsWithCipherSuites := []string{
+		binDir + "/etcd",
+		"grpc-proxy",
+		"start",
+		"--listen-cipher-suites", "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256",
+	}
+
+	argsWithoutCipherSuites := []string{
+		binDir + "/etcd",
+		"grpc-proxy",
+		"start",
+		"--listen-cipher-suites", "",
+	}
```

Comment:
Please use golang [subtest](https://go.dev/blog/subtests) to update the test case, which should can improve the readability.

Reply:
Thanks! This is fixed in the most recent update.